doc/nrf/libraries/bluetooth_services/mesh/light_ctrl_reg.rst

```diff
@@ -0,0 +1,20 @@
+.. _bt_mesh_light_ctrl_reg_readme:
+
+Light Lightness Control regulator
```

Comment:
Yes, we need to work on the rst files, obviously the names and references need to change.